test/tools/ossfuzz/abiV2Fuzzer.cpp

```diff
@@ -0,0 +1,57 @@
+/*
+	This file is part of solidity.
+
+	solidity is free software: you can redistribute it and/or modify
+	it under the terms of the GNU General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	solidity is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+	GNU General Public License for more details.
+
+	You should have received a copy of the GNU General Public License
+	along with solidity.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include <fstream>
+#include "abiV2FuzzerCommon.h"
+
+using namespace dev::test::abiv2fuzzer;
+using namespace std;
+using namespace dev;
+
+extern "C" int LLVMFuzzerTestOneInput(uint8_t const* _data, size_t _size)
```

Comment:
Why does this accept non-protobuf-encoded code?

Reply:
I started with a non proto fuzzer so that I could verify the test setup (and building the fuzzer as a single binary blob) even at a time when the proto test generation is not ready, hence this stub. 

Should I remove this stub from this PR altogether?

Reply:
No it's fine.